hs/app/reach/Main.hs

```diff
@@ -2450,6 +2451,149 @@ whoami = command "whoami" $ info f fullDesc
   where
     f = pure . script $ write [N.text| echo "$whoami'" |]
 
+type DeviceCode = Text
```

Comment:
Yuck

Reply:
Delete `type DeviceCode = Text` altogether?

Reply:
I believe [c9adb68](https://github.com/reach-sh/reach-lang/pull/762/commits/c9adb68673ce427cafc4f77dfc796425788e9b2f) addresses this.